deploy/operator.yaml

```diff
@@ -1982,13 +1982,13 @@ spec:
         - name: RELATED_IMAGE_RBAC_PROXY
           value: quay.io/brancz/kube-rbac-proxy:v0.11.0
         - name: RELATED_IMAGE_SELINUXD
-          value: quay.io/security-profiles-operator/selinuxd
+          value: quay.io/security-profiles-operator/selinuxd-fedora
         - name: OPERATOR_NAMESPACE
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: gcr.io/k8s-staging-sp-operator/security-profiles-operator:latest
-        imagePullPolicy: Always
+        image: localhost/security-profiles-operator:latest
+        imagePullPolicy: Never
```

Comment:
I think this is what is causing the `make verify-bundle` to break.

https://github.com/kubernetes-sigs/security-profiles-operator/runs/7564382381?check_suite_focus=true